### ISSUE TITLE
Cutover plan

### ISSUE THREAD

**Issue by the-maldridge** [void-linux/void-docs#104]
The docs have reached a pretty good level of maintenance, and I'm glad to see the high quality bar that has been maintained over here.  Lets start putting together a plan to once and for all EOL the wiki server.  I want to reclaim those resources and reassign them to putting a mirror back in Germany, something we desperately need since the German intranet is very sad as of late.

Lets target end of August for me to shut down wiki.voidlinux.org and reclaim the resource into the pool.  By that time lets grab any information that's worth saving.

@duncaen to assist in this process can you make the wiki R/O on August 1?

**Comment by nilium** [comment]
How hard is it going to be to do a dump of the wiki DB and share that?

**Comment by the-maldridge** [comment]
We can pull a dump of the DB pretty easily.  My original want had been to render out the entire site statically, but the mediawiki folks have informed me that there's no mechanism to do that.

**Comment by nilium** [comment]
Yeah. My goal is mostly to scan the dump for anything that's worth bringing over, since navigating the wiki's not very easy (at least for me), whereas grepping a dump is pretty simple.

**Comment by the-maldridge** [comment]
Ok, lets wait to go R/O and then we'll pull a dump.

**Comment by bobertlo** [comment]
I've been off doing other stuff but I'll keep an eye on this!

**Comment by lugubris** [comment]
We can also archive the page under Internet Archive before shutting down the wiki server. If you could generate a sitemap then there is a tool to do this automatically. 

-> https://github.com/agude/wayback-machine-archiver

**Comment by flexibeast** [comment]
i'd like to suggest that, before the wiki is shut down, Sitenotice (or `$wgSiteNotice`) is set to let wiki users know that the wiki is now deprecated, and that they should use https://docs.voidlinux.org/ instead. There have been comments on r/voidlinux that suggest to me that maybe a number of people still aren't aware of this ....

**Comment by bobertlo** [comment]
I checked and the internet archive coverage looks pretty good. We could crawl it if a webmaster would put up a sitemap.

Is there anything else necessary here? It seems like the move to the docs has gone well, but the wiki is still a good resource to the doc writers. Could we maybe put a banner or something in the header of the wiki saying that it is no longer maintained or updated and point people to the docs?

**Comment by flexibeast** [comment]
Blocked by #327.
